WireDomain/Sources/WireDomain/Account/StoredBackendEnvironment.swift

```diff
@@ -313,3 +291,24 @@ extension StoredBackendEnvironment.PinnedKey.Host {
         }
     }
 }
+
+struct StoredResolvedBackendMetadata: Codable, Sendable {
+
+    let apiVersion: APIVersion
+    let domain: String
+    let isFederationEnabled: Bool
+
+    enum APIVersion: UInt, Codable, Sendable {
+        case v0
+        case v1
+        case v2
+        case v3
+        case v4
+        case v5
+        case v6
+        case v7
+        case v8
+        case v9
+    }
```

Comment:
question: can't we use APIVersion defined in WireNetwork? updating the APIVersion will need to update this too

Reply:
removed